rsconnect/timeouts.py

```diff
@@ -0,0 +1,35 @@
+import os
+
+from rsconnect.exception import RSConnectException
+
+_CONNECT_REQUEST_TIMEOUT_KEY = "CONNECT_REQUEST_TIMEOUT"
+_CONNECT_REQUEST_TIMEOUT_DEFAULT_VALUE = "300"
+
+
+def get_timeout() -> int:
+    """Gets the timeout from the CONNECT_REQUEST_TIMEOUT env variable.
```

Comment:
should docs / the doc comment here note the unit of time being used for the timeout?